bittensor/synapse.py

```diff
@@ -542,33 +493,30 @@ def to_headers(self) -> dict:
         headers["header_size"] = str(sys.getsizeof(headers))
         headers["total_size"] = str(self.get_total_size())
         headers["computed_body_hash"] = self.body_hash
-        headers["hash_fields"] = base64.b64encode(
```

Comment:
does the axon need to know the other parsed hash_fields other then the required_hash_field?

Reply:
Nope, that is a hold over, nice catch! Will remove.